### ISSUE TITLE
Audit error: Required Accessibility gatherer encountered an error: Cannot read property 'push' of undefined

### ISSUE THREAD

**Issue by sorknes** [GoogleChrome/lighthouse#2484]
I don't know if this is the right place, but here it goes.

I'm having some issues with accessibility using lighthouse on a page I'm helping to build. All I get is this error:

> Audit error: Required Accessibility gatherer encountered an error: Cannot read property 'push' of undefined

I can't figure out why this is happening, and all help would be useful.

Thanks!

**Comment by ebidel** [comment]
Is there a URL you can share or steps to help us repro?

**Comment by paulirish** [comment]
@sorknes thanks for reporting. this is the right place. :)

seems like it might be specific to the URL you're testing. can you share that with us?


--------

the only `push` i see outside of axe code is this one:

```js
    const path = [];
    while (node && node.parentNode) {
      const index = getNodeIndex(node);
      path.push([index, node.nodeName]);
      node = node.parentNode;
    }
```

it is true that if `path` gets set to undefined, then the `path.push()` will fire this error...
though i'm not seeing how this would happen...

**Comment by mjpuser** [comment]
@ebidel aweber.com

**Comment by brendankenny** [comment]
closing for now since we've since updated our axe version. Please comment if you're still seeing an issue.
